8/Dockerfile

```diff
@@ -98,3 +98,30 @@ USER $user
 
 ENTRYPOINT [ "/docker-entrypoint.sh" ]
 CMD ["php-fpm"]
+
+FROM dist as development
```

Comment:
What do you think about calling this stage as the final docker image tag `dev`?

Reply:
fixed